app/controllers/users/reset_passwords_controller.rb

```diff
@@ -101,8 +100,8 @@ def handle_unsuccessful_password_reset(result)
     end
 
     def update_user
-      resource.update(confirmed_at: Time.current) unless resource.confirmed?
-      resource.update(password: user_params[:password])
+      resource.confirmed_at = Time.current unless resource.confirmed?
+      UpdateUser.new(user: resource, attributes: { password: user_params[:password] }).call
     end
```

Comment:
The comment about using one method to update the user, as opposed to mixing and matching setter + UpdateUser.